ingestion_tools/dataset_configs/template.yaml

```diff
@@ -1,137 +1,188 @@
+      ground_truth_used: OPTIONAL, STRING
+      precision: OPTIONAL IF ground_truth_used ELSE null, FLOAT ([0, 100])
+      recall: OPTIONAL IF ground_truth_used ELSE null, FLOAT ([0, 100])
+    annotation_method: REQUIRED, STRING
+    annotation_object: REQUIRED
+      description: OPTIONAL, STRING
+      id: REQUIRED, STRING (GOID)
+      name: REQUIRED, STRING
+      state: OPTIONAL, STRING
+    annotation_publications: OPTIONAL, STRING
+    annotation_software: OPTIONAL BUT RECOMMENDED, STRING
+    authors:  OPTIONAL BUT RECOMMENDED
+    - ORCID: OPTIONAL, STRING (ORCID)
+      corresponding_author_status: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+      name: REQUIRED, STRING
+      primary_author_status: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+    dates: REQUIRED
+      deposition_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+      last_modified_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+      release_date: REQUIRED, STRING, DATE FORMAT YYYY-MM-DD
+    ground_truth_status: REQUIRED, BOOLEAN (DEFAULT FALSE)
+    is_curator_recommended: OPTIONAL, BOOLEAN (DEFAULT FALSE)
+    method_type: REQUIRED, ENUM (manual, hybrid, automated)
+    version: OPTIONAL, STRING
+  sources: REQUIRED
+  - binning: OPTIONAL, INTEGER (DEFAULT 1) (Point, OrientedPoint, InstanceSegmentation)
+    columns: OPTIONAL, STRING (DEFAULT xyz) (Point)
+    delimiter: OPTIONAL, STRING (DEFAULT ',') (Point)
+    file_format: REQUIRED, STRING
+    filter_value: OPTIONAL, STRING (OrientedPoint, InstanceSegmentation)
+    glob_string: REQUIRED, STRING (REGEX)
+    is_visualization_default: OPTIONAL, BOOLEAN (DEFUALT false)
+    mask_label: OPTIONAL, INTEGER (DEFAULT 1) (SemanticSegmentationMask)
+    order: OPTIONAL, STRING (OrientedPoint, InstanceSegmentation)
+    shape: OPTIONAL, ENUM (InstanceSegmentation, OrientedPoint, Point, SegmentationMask, SemanticSegmentationMask)
+dataset_keyphotos: OPTIONAL
+- sources: REQUIRED
+  - literal: REQUIRED
+      value: REQUIRED
+        snapshot: REQUIRED, STRING
+        thumbnail: REQUIRED, STRING
+datasets: REQUIRED
+- metadata: REQUIRED
+    authors: SEE annotations.authors
+    cell_component: OPTIONAL
+      id: OPTIONAL, STRING (GOID)
+      name: REQUIRED, STRING
+    cell_strain: SEE cell_component
+    cell_type: SEE cell_component
+    cross_references: OPTIONAL
+      dataset_publications: OPTIONAL, STRING (DOIs)
+      related_database_entries: OPTIONAL, STRING (EMPIAR / EMDB IDs)
+    dataset_description: REQUIRED, STRING
+    dataset_identifier: REQUIRED, INTEGER
+    dataset_title: REQUIRED, STRING
+    dates: SEE annotations.dates
+    funding: OPTIONAL BUT RECOMMENDED
+    - funding_agency_name: OPTIONAL BUT RECOMMENDED, STRING
+      grant_id: OPTIONAL BUT RECOMMENDED, STRING
+    grid_preparation: OPTIONAL BUT RECOMMENDED, STRING
+    organism: OPTIONAL
+      name: REQUIRED, STRING
+      taxonomy_id: OPTIONAL, INTEGER
+    sample_preparation: OPTIONAL BUT RECOMMENDED, STRING
+    sample_type: REQUIRED, STRING
```

Comment:
Not a blocker, @uermel Can this also be a Enum of terms like  [cell, in_vitro, organism, virus, .. ]?

Reply:
Yes, it currently is implemented as an enum is in the schema! https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/e2c12eac9687b00a41a555dc98b71fabd80a1a4e/schema/v1.1.0/common.yaml#L601